### ISSUE TITLE
Prisma Integration

### ISSUE THREAD

**Issue by codejamninja** [nestjsx/crud#608]
How would I create a crud controller with Prisma?

**Comment by jkolyer** [comment]
I would look at porting the typeorm controller.  It's pretty large, so maybe you don't need everything all at once.  I may do this in a fork later, but right now I'm just learning nestjs.

**Comment by codejamninja** [comment]
I've already ported over a lot of it into the `nestjs-crud-prisma` package.

https://github.com/silicon-hills/nestjs-crud-prisma
https://www.npmjs.com/package/nestjs-crud-prisma

If anyone has any feedback, that would be great.

**Comment by eltonlau1994** [comment]
I find it's indeed useful and should we include in the documentation?

> I've already ported over a lot of it into the `nestjs-crud-prisma` package.
> 
> https://github.com/silicon-hills/nestjs-crud-prisma
> https://www.npmjs.com/package/nestjs-crud-prisma
> 
> If anyone has any feedback, that would be great.

**Comment by clayrisser** [comment]
Yeah I can definitely submit a PR. Give me a week to get things together for it.

**Comment by andrei9669** [comment]
just discovered prisma and it looks really interesting, would love to use it as an alternative to TypeORM.

**Comment by puneetverma05** [comment]
hey @clayrisser thanks for the port, any updates on the PR?
thanks

**Comment by technical-safrat** [comment]
Hi, Any update on prisma integration? this repo has been archived so any solution to integrate crud with prisma?
